app/controllers/bid_acceptances_controller.rb

```diff
@@ -1,9 +1,10 @@
 class BidAcceptancesController < ApplicationController
+	before_action :authenticate_broker_or_forward_freight!
```

Comment:
@rabingaire I added this method, we need to authenticate if there's a user, even if there's no visible user interface for it :)